src/widget/wcoverartmenu.cpp

```diff
@@ -70,10 +72,12 @@ void WCoverArtMenu::slotChange() {
 
     // TODO(rryan): Ask if user wants to copy the file.
 
+    QString selectedCoverExtension = QFileInfo(selectedCoverPath).suffix();
+
     CoverInfoRelative coverInfo;
     // Create a security token for the file.
     auto selectedCover = mixxx::FileAccess(mixxx::FileInfo(selectedCoverPath));
-    QImage image(selectedCoverPath);
+    ImageFileData image = ImageFileData::fromFilePath(selectedCoverPath);
```

Comment:
This is a file access in the main thread ... not sure if we want to remove that?

Reply:
We must delete that IMHO, according to my tests i come across with freezing in small amount of time. I am not sure what would be the solution for this. My first assumption is:

Instead of getting ImageFileData on here, we can leave it as QImage. Then we can pass the 'location' of the cover art instead of the "QImage or ImageFileData" to the CoverArtCopyWizard. Then we can call the ImageFileData on the worker via location.

What do you think?

Reply:
QImage does also the disc access. Our new class should not have a performance regression, has it? 
For now the code can remain, just add a TODO that this should be done in a thread.

Reply:
> QImage does also the disc access. 

Hmm maybe that was like this before we have ImageFileData, I can not remember.

> Our new class should not have a performance regression, has it? 

I have tried with the QImage aswell, didn't see any difference.

> For now the code can remain, just add a TODO that this should be done in a thread.

👍